inc/extras.php

```diff
@@ -69,3 +69,220 @@ function llorix_one_lite_render_title() {
 	}
 	add_action( 'wp_head', 'llorix_one_lite_render_title' );
 endif;
+
+
+if(!function_exists('llorix_one_lite_pll')){
+	/**
+	 * Filter to apply translations if polylang plugin is installed.
+	 * @param string $input Input to translate.
+	 *
+	 * @return string
+	 */
+	function llorix_one_lite_pll($input){
+		if(function_exists('pll__')){
+			return pll__($input);
+		}
+		return $input;
+	}
+	add_filter('llorix_one_lite_language_filter','llorix_one_lite_pll');
```

Comment:
I'm not sure, but shouldn't this add_filter be outsite the if ? Maybe inside an if that checks that the llorix_one_lite_pll exists